client/components/gravatar/index.jsx

```diff
@@ -64,10 +70,25 @@ module.exports = React.createClass( {
 		}
 
 		const alt = this.props.alt || this.props.user.display_name;
-		const avatarURL = this.getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
+
+		let avatarURL = '';
+		if ( this.props.user.ID &&
```

Comment:
Updated in 29ab6b3 - using `getUserTempGravatar` to get the current user temp Gravatar, if it exists